.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           name: Setup base environment variable
           command: |
             echo "export BUILD_SHA=$CIRCLE_SHA1" >> $BASH_ENV
-            echo "export SSH_FILE_FOR_SECRETS=~/.ssh/id_rsa_050e96bc58f960ccb634cbf505912c11" >> $BASH_ENV
+            echo "export SSH_FILE_FOR_SECRETS=~/.ssh/id_rsa_65357fbcc65a92dfff23ac166c2e76de" >> $BASH_ENV
       - run: &deploy_scripts
           name: cloning deploy scripts
           command: 'git clone git@github.com:ministryofjustice/fb-deploy.git deploy-scripts'
```

Comment:
This is also just referencing fb-deploy right? Do doesn't need an SSH key or fingerprint, just the BUILD_SHA variable

Reply:
this is need to clone the fb-publisher-deploy repo